src/lv_interop.cc

```diff
@@ -168,8 +184,13 @@ namespace grpc_labview
             return std::string();
         }
 
-        auto count = (*lvString)->cnt;
-        auto chars = (*lvString)->str;
+        LStrHandle utf8String = nullptr;
+        auto error = ConvertSystemStringToUTF8(lvString, &utf8String);
+        if (error != 0) throw std::runtime_error("Failed to convert string encoding.");
```

Comment:
FYI, nothing is currently catching these exceptions, which is also true for the existing std::string usage. I will try fixing https://github.com/ni/grpc-labview/issues/438 in a separate PR.